client/components/Users/UserOverview.jsx

```diff
@@ -15,58 +16,78 @@ export default class UserOverview extends React.Component {
     userFields: React.PropTypes.array.isRequired
   }
 
+  constructor(props) {
+    super(props);
+
+    const searchUserFields = _.filter(this.props.userFields, { search: true });
+    this.searchOptions = _.map(searchUserFields, (field, index) => {
+      return {
+        title: field.label,
+        value: field.property,
+        filterBy: field.property,
+        selected: index === 0
+      };
+    });
+
+    const selectedFilter = _.find(this.searchOptions, { selected: true });
+    this.state = { selectedFilter };
+
+    this.renderSearchBar.bind(this);
+    this.onKeyPress = this.onKeyPress.bind(this);
+    this.onReset = this.onReset.bind(this);
+    this.onHandleOptionChange = this.onHandleOptionChange.bind(this);
+  }
+
   onKeyPress = (e) => {
     if (e.key === 'Enter') {
-      this.props.onSearch(findDOMNode(this.refs.search).value);
+      e.preventDefault();
+      const query = `${this.state.selectedFilter.filterBy}:${e.target.value}*`;
+      this.props.onSearch(query);
```

Comment:
I'm not convinced that we should be adding the star (*), maybe we should just make the user add the star.